sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -0,0 +1,266 @@
+    "incident_wavelength/@units": "@attrs:metadata/scientificMetadata/laser/wavelength/unit",
+    "incident_energy": "@attrs:metadata/scientificMetadata/laser/incident_energy/value",
+    "incident_energy/@units": "@attrs:metadata/scientificMetadata/laser/incident_energy/unit",
+    "pulse_duration": "@attrs:metadata/scientificMetadata/laser/pulse_duration/value",
+    "pulse_duration/@units": "@attrs:metadata/scientificMetadata/laser/pulse_duration/unit",
+    "incident_polarization": "@attrs:metadata/scientificMetadata/laser/incident_polarization/value",
+    "incident_polarization/@units": "@attrs:metadata/scientificMetadata/laser/incident_polarization/unit",
+    "pulse_energy": "@attrs:metadata/scientificMetadata/laser/pulse_energy/value",
+    "pulse_energy/@units": "@attrs:metadata/scientificMetadata/laser/pulse_energy/unit",
+    "associated_source": "/entry/instrument/source_pump"
+  },
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MANIPULATOR[manipulator]": {
+    "type": "@attrs:metadata/scientificMetadata/manipulator/type",
+    "temperature_sensor": {
+      "name": "sample_temperature",
+      "measurement": "temperature",
+      "value": "@attrs:metadata/scientificMetadata/sample/sample_temperature/value",
+      "value/@units": "@attrs:metadata/scientificMetadata/sample/sample_temperature/unit"
+    },
+    "sample_bias_voltmeter": {
+      "name": "sample_bias",
+      "measurement": "voltage",
+      "value": "@attrs:metadata/scientificMetadata/manipulator/sample_bias/value",
+      "value/@units": "@attrs:metadata/scientificMetadata/manipulator/sample_bias/unit"
+    }
+  },
+  "/ENTRY[entry]/SAMPLE[sample]": {
+    "description": "@attrs:metadata/scientificMetadata/sample/sample_chemical_name",
+    "name": "@attrs:metadata/scientificMetadata/sample/sample_name",
+    "situation": "vacuum",
+    "preparation_description": "@attrs:metadata/scientificMetadata/sample/preparation_description",
+    "temperature": {
+      "temperature_sensor": "@link:/entry/instrument/manipulator/temperature_sensor"
+    },
+    "gas_pressure": {
+      "pressure_gauge": "@link:/entry/instrument/pressure_gauge"
+    },
+    "bias": {
+      "voltmeter": "@link:/entry/instrument/manipulator/sample_bias_voltmeter"
+    }
+  },
+  "/ENTRY[entry]/PROCESS[process]/DISTORTION[distortion]": {
+    "symmetry": "@attrs:metadata/momentum_correction/correction/rotsym",
+    "original_centre": "@attrs:metadata/momentum_correction/correction/pcent",
+    "original_points": "@attrs:metadata/momentum_correction/correction/pouter",
+    "cdeform_field": "@attrs:metadata/momentum_correction/correction/cdeform_field",
+    "rdeform_field": "@attrs:metadata/momentum_correction/correction/rdeform_field"
+  },
+  "/ENTRY[entry]/PROCESS[process]/REGISTRATION[registration]": {
+    "depends_on": "@attrs:metadata/momentum_correction/registration/depends_on",
+    "TRANSFORMATIONS[transformations]": {
```

Comment:
the spelling here of transformations was tranformations so fixed that too @kutnyakhov